cfn-resources/integration-tests/project/project_test.go

```diff
@@ -0,0 +1,237 @@
+// Copyright 2023 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package project
```

Comment:
I think that the PR is missing a github action that runs the code. Are you planning to add it in a follow-up PR? if yes, could you share the log of a successful run of the test? 

I am happy to create a ticket to add the GitHub action to be completed once we have completed the current project.

Reply:
Yes that will be addressed in a follow up PR.

Created ticket: [INTMDB-722](https://jira.mongodb.org/browse/INTMDB-722)

Here's the log of a successful run of the test:
```
/usr/local/go/bin/go tool test2json -t /private/var/folders/yg/18z6bwv522j1lg_7sffg1nvw0000gp/T/GoLand/___TestProjectCFN_in_github_com_mongodb_mongodbatlas_cloudformation_resources_integration_tests_project.test -test.v -test.paniconexit0 -test.run ^\QTestProjectCFN\E$
=== RUN   TestProjectCFN
    project_test.go:65: Setting up suite
    project_test.go:95: Setting clients
    util.go:53: New E2E test resource type MongoDB::Atlas::Project770 successfully published to private registry!
    project_test.go:199: Setting up prerequisites
=== CONT  TestProjectCFN
    project_test.go:191: Cleaning up prerequisites
    util.go:41: E2E test resource type MongoDB::Atlas::Project770 successfully de-registered from private registry!
--- PASS: TestProjectCFN (139.95s)
=== RUN   TestProjectCFN/Validate_Template
    --- PASS: TestProjectCFN/Validate_Template (0.58s)
=== RUN   TestProjectCFN/Create_Stack
    --- PASS: TestProjectCFN/Create_Stack (10.58s)
=== RUN   TestProjectCFN/Update_Stack
    --- PASS: TestProjectCFN/Update_Stack (17.02s)
=== RUN   TestProjectCFN/Delete_Stack
    --- PASS: TestProjectCFN/Delete_Stack (7.18s)
PASS

Process finished with the exit code 0
```